assets/src/edit-story/app/api/getAllPageLayouts.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import getAllTemplates from '../../../dashboard/templates';
+
+const replacePageImagesWithPlaceholders = (page) => {
+  return {
+    ...page,
+    elements: page.elements.map((element) => {
+      if (element.type !== 'image') {
+        return element;
+      }
+
+      return {
+        ...element,
+        focalX: 50,
+        focalY: 50,
+        backgroundColor: {
+          color: { r: 219, g: 223, b: 226 },
+        },
+        resource: {
+          type: 'image',
+          mimeType: 'image/png',
+          // TODO: replace this with `${cdnURL}/path/to/image.jpg`
+          src: 'http://www.squidsuds.com/i/gwsplaceholder.png',
```

Comment:
Since it's likely going to be a very small image, it can totally live under `assets/images/editor` instead. Just need to provide `assetsURL` in `Story_Post_Type.php` like we do for dashboard:

https://github.com/google/web-stories-wp/blob/13f1a40966d5213ba7ec6832777b317d84a29e8e/includes/Dashboard.php#L344

Then you can do

```js
const { assetsURL } = useConfig();
// ...

src: `${assetsURL}images/editor/grid-placeholder.png`,
```